torch/_utils.py

```diff
@@ -52,71 +52,40 @@ def _type(self, dtype=None, non_blocking=False, **kwargs):
     return dtype(self.size()).copy_(self, non_blocking)
 
 
-def _hpu(self, device=None, non_blocking=False, **kwargs):
-    """Returns a copy of this object in HPU memory.
+def _backend(self, backend_name, device=None, non_blocking=False, **kwargs):
+    """Returns a copy of this object in backend memory.
 
-    If this object is already in HPU memory and on the correct device, then
+    If this object is already in backend memory and on the correct device, then
     no copy is performed and the original object is returned.
 
     Args:
-        device (int): The destination HPU id. Defaults to the current device.
```

Comment:
update the doc

Reply:
I changed the doc. You probable accidentally looked into prev. version.